src/rebar_file_utils.erl

```diff
@@ -142,7 +144,7 @@ mv(Source, Dest) ->
     end.
 
 win32_robocopy_ok({ok, _}) -> true;
-win32_robocopy_ok({error, {Rc, _}}) when Rc<9, Rc==16 -> true;
+win32_robocopy_ok({error, {Rc, _}}) when Rc<9; Rc=:=16 -> true;
```

Comment:
You had code in sh_loop that resolved this that I removed now when we have this function.